src/Applications/Applications/examples/v1.0/Add-MgApplicationKey.md

```diff
@@ -1,18 +1,30 @@
-### Example 1: {{ Add title here }}
-```powershell
-PS C:\> {{ Add code here }}
-
-{{ Add output here }}
+###Example 1
 ```
```

Comment:
Code blocks are missing syntax highlighting for powershell.
```suggestion
``` powershell
```

Reply:
I was asked not to include powershell

Reply:
I can include it again :-)